<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GestionUsuarios</class>
 <widget class="QMainWindow" name="GestionUsuarios">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gesti√≥n de Usuarios</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="nombreLabel">
        <property name="text">
         <string>Nombre:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="nombreLineEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="buttonsLayout">
      <item>
       <widget class="QPushButton" name="crearPushButton">
        <property name="text">
         <string>Crear</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableView" name="usuariosTableView"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="editarEliminarLayout">
      <item>
       <widget class="QPushButton" name="editarPushButton">
        <property name="text">
         <string>Editar Usuario</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="eliminarPushButton">
        <property name="text">
         <string>Eliminar Usuario</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
